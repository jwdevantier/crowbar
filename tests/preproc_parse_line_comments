// <<crowbar
// counter = 0
// @component
// def hello(emit):
//     global counter
//     counter += 1
//     emit(f"hello{counter}")
// >>
// <<end>>

c-style line comments:
// <<crowbar
// emit(hello())
// >>
// <<end>>

python-style line comments:
# <<crowbar
# emit(hello())
# >>
# <<end>>

lua-style line comments:
-- <<crowbar
-- emit(hello())
-- >>
-- <<end>>
