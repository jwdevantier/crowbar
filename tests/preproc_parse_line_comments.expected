// <<crowbar
// counter = 0
// @component
// def hello(emit):
//     global counter
//     counter += 1
//     emit(f"hello{counter}")
// >>
// <<end>>

c-style line comments:
// <<crowbar
// emit(hello())
// >>
hello1
// <<end>>

python-style line comments:
# <<crowbar
# emit(hello())
# >>
hello2
# <<end>>

lua-style line comments:
-- <<crowbar
-- emit(hello())
-- >>
hello3
-- <<end>>
